### ISSUE TITLE
issuing "--build" not working

### ISSUE THREAD

**Issue by darlov** [cake-contrib/Cake.CMake#19]
From @rpriest1260
Hello, I am wondering if others are experiencing the same issue, that I am seeing.
I can't seem to execute the **--build** command properly using the Cake.CMakeBuild (I also see the same thing with the Cake.Cmake addin). 
The issue is, that in both of these addins, the source directory is prepended on every command.  So, when you specify the  **_"--build %dir% "_** argument,  it doesn't respect the command and it just regenerates the Cache files. At first, I thought it was the quoted argument strings, but I think it is just the prepending of the source directory.

If I am misunderstanding how the build command should be run, please advise.

Otherwise should another argument be added in the CMakeSettings to specify "Build", and thus do not prepend the Source directory to the command? 

I thought I would ask if I am missing something, before submitting a pull request. 

Thoughts?

**My setup:**
Windows 10
Cake.CMakeBuild 0.3.0
CMake-win64.3.12.1

Here is how I have the Task structured, right now:

```
Task("Build")
    .IsDependentOn("Cache")
    .Does(() =>
{
    Information("Building Api");    
    List<string> options = new List<string>();
    options.Add(string.Format("--build", MakeAbsolute(GetWorkingDirectory()).FullPath));
    options.Add(string.Format(MakeAbsolute(GetWorkingDirectory()).FullPath));  
    CMake(GetWorkingDirectory(), new CMakeSettings() 
        {
        Options = options,
        WorkingDirectory = GetWorkingDirectory()
        });
});

```

The output that task (when setting Verbosity=Diagnostic is:

```
========================================
Build
========================================
Executing task: Build
Building Api
Executing: "Z:/test/cake/cmakebuild/tools/Addins/CMake-win64.3.12.1/bin/cmake.exe" "Z:/test/cake/cmakebuild/build/out/x64/Debug" "--build" "Z:/test/cake/cmakebuild/build/out/x64/Debug"
-- Configuring done
-- Generating done
-- Build files have been written to: Z:/test/cake/cmakebuild/build/out/x64/Debug
Finished executing task: Build

```

Which again, is just regenerating the cache files.

However, ever if I cut the command that is generated, remove the prepended directory, and just issue the following at the command line:

`Z:\test\cake\cmakebuild>"Z:/test/cake/cmakebuild/tools/Addins/CMake-win64.3.12.1/bin/cmake.exe" "--build" "Z:/test/cake/cmakebuild/build/out/x64/Debug"`

I get the desired result of it building my sample (which is just the Tutorial sample from the CMake website -  https://cmake.org/cmake-tutorial/ ) :

`[2/2] Linking CXX executable Tutorial.exe`

**Comment by darlov** [comment]
Hello @rpriest1260
The pre-release version has been released with fix of this issue. For more details you can see the [README.md](https://github.com/cake-contrib/Cake.CMake/blob/develop/README.md) and [Release 1.1.0-beta0001](https://github.com/cake-contrib/Cake.CMake/releases/tag/1.1.0-beta01). 
Thanks!

**Comment by darlov** [comment]
Fixed in the 1.1.1 version.

**Comment by rpriest1260** [comment]
@moongdev , sorry for the late response. I got caught up with other projects. But I tried your changes in version 1.1.1. and they worked. Thanks.
